00-spatio-temporal-science-questions.Rmd

```diff
@@ -225,214 +187,209 @@ achieve this goal we need data that:
 {: .notice}
 
-
 ### NDVI Measures Plant Greenness
 
-We can use remote sensing imagery to
-derive the Normalized Difference Vegetation Index (NDVI), a quantitative index
-that measures how "green" or healthy vegetation is on the ground. NDVI greenness 
-values ranging from 0-1 where 0 represents minimal or no greenness
-and 1 represents maximum greenness. NDVI is an ideal data product to use to estimate
-the timing of plant "green-up" and "brown-down" over large areas.
+We can use remote sensing imagery to derive the Normalized Difference Vegetation
+Index (NDVI) - a quantitative index that measures how "green" or healthy
+vegetation is on the ground. NDVI greenness values ranging from 0-1 where 0 
+represents minimal or no greenness and 1 represents maximum greenness. NDVI is 
+an ideal data product used to estimate the timing of plant "green-up" and 
+"brown-down" over large areas.
 
 <figure>
  <a href="http://earthobservatory.nasa.gov/Features/MeasuringVegetation/Images/ndvi_example.jpg"> 
  <img src="http://earthobservatory.nasa.gov/Features/MeasuringVegetation/Images/ndvi_example.jpg"></a>
-    <figcaption>NDVI is calculated from the visible and near-infrared light
+    <figcaption> NDVI is calculated from the visible and near-infrared light
     reflected by vegetation. Healthy vegetation (left) absorbs most of the
     visible light that hits it, and reflects a large portion of
     near-infrared light. Unhealthy or sparse vegetation (right) reflects more
-    visible light and less near-infrared light. Image & Caption Source: NASA 
+    visible light and less near-infrared light. Source: NASA 
     </figcaption>
 </figure>
 
-* <a href="http://earthobservatory.nasa.gov/Features/MeasuringVegetation/measuring_vegetation_2.php" target="_blank">
-More on NDVI from NASA</a>
+We can visualize NDVI data and see the change in color (representing the change 
+in the index) as the growing season progresses across the year. In this figure,
+each plot presents a snapshot of the NDVI values across an area of 120m x 150m. 
 
+<figure>
+	<a href="{{ site.baseurl }}/images/dc-spatio-temporal-intro/NDVI-Landsat-RS.png">
+	<img src="{{ site.baseurl }}/images/dc-spatio-temporal-intro/NDVI-Landsat-RS.png"></a>
+	<figcaption> Remote sensing data can be used to measure the greening and
+	browning of plants over larger areas. Because these data are often collected
+	repeatedly using satellites orbiting the earth, it provides a nice time series
+	to use for many research questions. 
+	Source: National Ecological Observatory Network (NEON) figure based on Landsat
+	Data.
+	</figcaption>
+</figure>
+
+* <a href="http://earthobservatory.nasa.gov/Features/MeasuringVegetation/measuring_vegetation_2.php" target="_blank"> More on NDVI from NASA</a>
 
-<i class="fa fa-star"></i> **Data Note:** NDVI can be calculated using any image
+<i class="fa fa-star"></i> **Data Tip:** NDVI can be calculated using any image
```

Comment:
@mjones01  I've been using data note for anything that's not coding related. Zack suggested it and i thought it was a good suggestion. Please don't change this.